upgrade-notes/backend_20250127_145003.md

```
@@ -0,0 +1,9 @@
+#### new B2B customer user roles for cart and order access and manipulation ([#3756](https://github.com/shopsys/shopsys/pull/3756))
+
+- [features moved](#movement-of-features-from-project-base-to-packages) from project-base to the frontend-api package:
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)